docs/range-tokens/deployment-tutorial.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Range Tokens Deployment Tutorial
+sidebar_label: Deployment Tutorial
+---
+
+The process to launch and manage a range token on UMA is surprisingly simple. This tutorial will show you how to deploy a range token using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository. By following these instructions, you will be able to launch a custom LSP on Kovan testnet and Ethereum mainnet.
```

Comment:
```suggestion
The process to launch and manage a range token on UMA is surprisingly simple. This tutorial will show you how to deploy a range token using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository.
```

Reply:
Updated and made sure the change was also included in the success token section.